DesignSystem/Sources/DesignSystem/BrandingManager+Extensions.swift

```diff
@@ -87,6 +90,9 @@ public extension Typography.Key {
     static let xlarge   = Typography.Key("xlarge")
     static let xxlarge  = Typography.Key("xxlarge")
     static let xxxlarge = Typography.Key("xxxlarge")
+    static let x4large  = Typography.Key("x4large")
+    static let x5large  = Typography.Key("x4large")
+    static let x6large  = Typography.Key("x4large")
```

Comment:
These shouldn't be on this level.. you can extend them in any way you like in your own brand but this is too deep imo.
I'd prefer you extend `Typography.Key` in your own project and add these. Then it's not exposed to other brands / projects as a default. Also the key is `"x4large"` for all 3. you'll need to change the bottom 2.